### ISSUE TITLE
Possibly an issue with stacktrace middleware

### ISSUE THREAD

**Issue by aaronc** [clojure-emacs/cider-nrepl#214]
I'm getting seeing this stacktrace in the console: https://gist.github.com/aaronc/a257d737aef922366a5f and cider-nrepl reports the following:

```
Stack trace of root exception is empty; this is likely due to a JVM optimization that can be disabled with -XX:-OmitStackTraceInFastThrow.
java.lang.NullPointerException: 
```
I tried setting this JVM option but nothing changes. Is this possibly an issue with cider-nrepl's stacktrace middleware. Maybe in `flag-tooling`?

The strange thing is that when I disable cider-nrepl, I still get the same error message in cider but no stacktrace in the console. Seems like a tricky case...

**Comment by bbatsov** [comment]
Can you provide us with some simple repro steps for this issue?

**Comment by aaronc** [comment]
Honestly I think it would be pretty hard at this point. I believe we found a way to debug this error even with no stacktrace and it was over a month ago. But what I did notice is that the stacktrace is for a `NullPointerException` generated from the regex in `flag-tooling` - probably just an edge case where `(:name %)` returns `nil` on line 68 in stacktrace.clj. This clearly doesn't happen very often, but it might be nice to just check for `nil` there before running the regex and then the exception at least wouldn't be cider's fault.
